browser/ably-prologue.js

```diff
@@ -1,2 +1,12 @@
 (function() {
 	window.Ably = {};
+
+  /*
+    Prevent libraries such as CryptoJS plugging into AMD or CommonJS
+    as the libraries loaded are expected in the `this` context.
+    `require` is only used within the Node.js library, the ably-js browser library
+    is built as a single Javascript file.
+  */
+  var define = undefined,
+      exports = undefined,
+      require = undefined;
```

Comment:
Also, just using `var define` without the undefined should be enough to have this variable within the scope as opposed to the global scope.